app/views/state_file/questions/terms_and_conditions/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title = t(".title") %>
+<% content_for :page_title, title %>
+<% content_for :card do %>
+  <div>
+    <h1 class="h2" id="main-question"><%= title %></h1>
+    <% if params[:us_state] == "ny" %>
+      <p><%=t(".ny_text_html") %></p>
+    <% end %>
+    <p><%=t(".body_text_html") %></p>
+  </div>
+  <p><%=t(".read_text") %></p>
+  <%= link_to next_path, class: "button button--primary", id: "firstCta" do %>
+    <%= t(".accept") %>
+  <% end %>
+  <div><button class="button">
+    <%= t(".decline") %>
+  </button></div>
```

Comment:
What is supposed to happen if they decline?

Reply:
Thats a separate story I'm working on right now: https://www.pivotaltracker.com/story/show/186581793